packages/cubejs-query-orchestrator/test/unit/QueryOrchestrator.test.js

```diff
@@ -1575,4 +1575,14 @@ describe('QueryOrchestrator', () => {
     await Promise.all(promises);
     expect(mockDriver.tables).toContainEqual(expect.stringMatching(/orders_delay/));
   });
+
+  test('fetch table schema', async () => {
+    const queryCacheSpy = jest.spyOn(queryOrchestrator.queryCache, 'fetchSchema')
+      .mockImplementation(() => '');
+      
+    await queryOrchestrator.fetchSchema('default');
+
+    expect(queryCacheSpy.mock.calls.length).toEqual(1);
```

Comment:
Just check calling fetchSchema method inside.

Reply:
Yeah, I see it. But it would be great if you will try to test `QueryOrchestrator <-> QueryCache` integration here instead. This test is senseless.